internal/test/utils.go

```diff
@@ -24,3 +26,12 @@ func ExpectExitCoderError(t *testing.T, err error) {
 		t.Error("expected error to fulfill cli.ExitCoder interface, but does not")
 	}
 }
+
+// ValidateFprint is a test utility to validate the returned error of
+// fmt.Fprint.
+func ValidateFprint(t *testing.T, w io.Writer, a interface{}) {
```

Comment:
this is okay, but this utility isn't just validating Fprint, its also calling it. Just seems weird to call `ValidateFprint()` on something that isn't the Fprint output, if that makes sense.

You could just rename this to `Fprint` -- then when its called, its `test.Fprint` which makes it clear that its calling Fprint and will just handle the checking logic